core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -129,6 +137,33 @@ public void minDocFreq(float minDocFreq) {
         this.minDocFreq = minDocFreq;
     }
 
+    public DirectSpellChecker getDirectSpellChecker() {
```

Comment:
I changed this from static to be a method in DirectSpellcheckerSettings because before we used to pass in an instance of DirectSpellcheckerSettings. Now we call the non-static method on that instance. I don't have strong preferences, but I think of this more like a "build" method, that is creating a lucene object from existing settings. Makes sense or should I change this back?